packages/server/__snapshots__/6_visit_spec.coffee.js

```diff
@@ -507,7 +507,7 @@ exports['e2e visit normal response timeouts fails when visit times out 1'] = `
 
 Your page did not fire its 'load' event within '1000ms'.
 
-You can try increasing the 'pageLoadTimeout' value in 'cypress.json' to wait longer.
+You can try increasing the 'pageLoadTimeout' value in your configuration file to wait longer.
```

Comment:
I'm not sure how I feel about making these errors more generic. This leads to more "dead-end" behavior.

You did a lot of work in the **other** packages to provide the right value here, yet I feel like it's actually **more** important to do that when displaying errors... especially in `cypress run` mode.